src/buildHTML.js

```diff
@@ -997,6 +995,11 @@ var groupTypes = {
         return makeSpan([options.style.reset(), newStyle.cls()], inner);
     },
 
+    font: function(group, options, prev) {
+        var font = group.value.font;
+        return buildGroup(group.value.body, options.withFont(font), prev);
```

Comment:
I don't think I'll be able to test it without this.  I'll pull the changes to options.js in instead.